cli/src/pcluster/config/cluster_config.py

```diff
@@ -1254,7 +1255,7 @@ def _register_storage_validators(self):
                         new_storage_count["efs"] += 1
             self._register_validator(
                 ExistingFsxNetworkingValidator,
```

Comment:
Minor: Please keep track in our backlog that we must cover these kind of changes in unit tests like `test_validators_are_called_with_correct_argument` to be sure that expected validators are called with the correct arguments